hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java

```diff
@@ -381,6 +388,10 @@ private void updateKeys(
             validateTrue(originalToken.hasPauseKey(), TOKEN_HAS_NO_PAUSE_KEY);
             builder.pauseKey(op.pauseKey());
         }
+        if (op.hasMetadataKey()) {
+            validateTrue(originalToken.hasMetadataKey(), TOKEN_HAS_NO_METADATA_KEY);
+            builder.metadataKey(op.metadataKey());
```

Comment:
`OrThrow()` here (to match `op.metadataOrThrow()`)? 
`builder.metadataKey(op.metadataKeyOrThrow())`

Reply:
Removing metadataOrThrow() and orThrow() as well so we keep it consistent with the other keys / code